demisto_sdk/commands/common/tests/integration_test.py

```diff
@@ -414,10 +414,24 @@ def test_invalid_feed(self, feed, fromversion):
     def test_is_fetch_params_exist(self):
```

Comment:
separate into different tests, document the scenario for each test (sanity, missing field, malformed field).
in case of error make sure a call to `print_error` is made